internal/security/secrets/certificates/tls.go

```diff
@@ -130,19 +132,19 @@ func (gen tlsCertGenerator) Generate() (err error) {
 		return fmt.Errorf("failed to parse TLS server certificate - DER: %s", err.Error())
 	}
 
-	gen.logger.Debug(fmt.Sprintf("Saving TLS server private key to PEM file: %s", gen.seed.TLSKeyFile))
+	gen.logger.Debug(fmt.Sprintf("Saving TLS server private key to PEM file: %s", gen.certificateSeed.TLSKeyFile))
 	skPKCS8, err := x509.MarshalPKCS8PrivateKey(privateTLS)
 	if err != nil {
 		return fmt.Errorf("failed to encode TLS server key: %s", err.Error())
 	}
 
-	err = gen.writer.Write(gen.seed.TLSKeyFile, pem.EncodeToMemory(&pem.Block{Type: "PRIVATE KEY", Bytes: skPKCS8}), 0600)
+	err = gen.writer.Write(gen.certificateSeed.TLSKeyFile, pem.EncodeToMemory(&pem.Block{Type: "PRIVATE KEY", Bytes: skPKCS8}), 0600)
 	if err != nil {
 		return fmt.Errorf("failed to save TLS server private key: %s", err.Error())
 	}
 
-	gen.logger.Debug(fmt.Sprintf("Saving Root CA certificate to PEM file: %s", gen.seed.CACertFile))
-	err = gen.writer.Write(gen.seed.TLSCertFile, pem.EncodeToMemory(&pem.Block{Type: "CERTIFICATE", Bytes: tlsDER}), 0644)
+	gen.logger.Debug(fmt.Sprintf("Saving TLS server certificate to PEM file: %s", gen.certificateSeed.CACertFile))
```

Comment:
the string to log should be `gen.certificatedSeed.TLSCertFile`, not `gen.certificateSeed.CACertFile`

Reply:
Updated.